src/components/RestaurantBox.js

```diff
@@ -3,9 +3,18 @@ import restaurantImgOne from "../img/restaurant1.png";
 import { ReactComponent as Clock } from "../icons/clock.svg";
 import { ReactComponent as Tray } from "../icons/tray.svg";
 import { ReactComponent as Pin } from "../icons/pin.svg";
-import { ReactComponent as Heart } from "../icons/heart.svg";
+import { ReactComponent as HeartFull } from "../icons/heart-full.svg";
+import { ReactComponent as HeartEmpty } from "../icons/heart-empty.svg";
+
+export default function RestaurantBox({ name, cuisine, area, bookmarked }) {
+  function handleBookmarked() {
```

Comment:
We normally use `handle...` functions to handle events. 
In this case, this function actually renders the bookmark. I'd call it `renderBookmarked` or something similar